test/e2e/resize_nodes.go

```diff
@@ -341,7 +428,8 @@ var _ = Describe("Nodes", func() {
 			}
 		})
 
-		testName = "should survive network partition."
+		// TODO: this test has nothing to do with resizing nodes so it should be moved elsewhere.
+		testName = "should survive unresponsive node."
```

Comment:
My apologies, I'm wrong.  You do also check that the node comes back again and can host a pod.

In that case I'd suggest splitting this into two separate tests:

1. testing that pods on the disconnected node are rescheduled, and
2. testing that a disconnected node, once reconnected, can host a pod.

You could do this in a separate PR I guess.